README.md

```diff
@@ -1,50 +1,62 @@
-_This document is intended to be a template for working groups established by the TAC, to be used as the README.md file in its repository. All items in italics are descriptions of what context should be there; it is intended to be replaced by content. It is modeled off the [CNCF Serverless WG](https://github.com/cncf/wg-serverless)._
+# Python 3 Working Group
 
-_The intention is that each WG has its own GitHub repo under the [ASWF organization](https://github.com/AcademySoftwareFoundation)._
 
-# _Working group name_
+The purpose of this Working Group is to ease the transition from Python 2 to Python 3 for Academy Software Foundation (ASWF) projects and ASWF members’ internal codebases. We hope to achieve this by coordinating key milestones in the VFX industry’s transition and facilitating the sharing of expertise between ASWF members.
 
-_Describe the purpose of the group in no more that 4-5 sentences._
 
-_Goals are the distinct outcomes that is to be anticipated from the working group, serving as a method for validating activities._
+The goals of the Working Group are the distinct outcomes that are to be anticipated from the Working Group. They are as follows:
 
-Goals of the WG are:
 
-1. _goal 1_
-2. ...
+* Share and develop best practices for transitioning to Python 3.
+* Coordinate transition efforts between software developers and vendors.
+* Articulate and share common experiences to avoid duplicate efforts.
 
-_Non-goals are things that the working group are intentionally choosing not to do, the things not expected to change due to the working groups efforts, or the subject matter areas that the working groups doesn’t want to address at this time. _
 
-Non-goals of the WG are:
+The non-goals of the Working Group are areas that the working group are intentionally choosing not to address. They are as follows:
 
-1. _non goal 1_
-2. ...
 
-The TAC member sponsor of this working group is _TAC sponsor name_
+* To provide resources for helping projects transition to Python 3.
```

Comment:
Comment from the TAC meeting: Let's be more specific here and make it clear that we mean engineering resources.